test/import.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var tap = require('tap');
+var spawn = require('child_process').spawn;
+var concat = require('concat-stream');
+
+tap.test('importing a single module', function (t) {
+    t.plan(2);
+
+    var tc = function (rows) {
+        t.same(rows.toString('utf8'), [
+            'TAP version 13',
+            '# module-a',
+            'ok 1 loaded module a',
+            '# test-a',
+            'ok 2 module-a loaded in same context',
+            'ok 3 test ran after module-a was loaded',
+            '',
+            '1..3',
+            '# tests 3',
+            '# pass  3',
+            '',
+            '# ok'
+        ].join('\n') + '\n\n');
+    };
+
+    var ps = tape('-r ./import/a import/test-a.js');
```

Comment:
`import/test-a.js` does not exists, it should be `import/test-a.mjs` right or is it supposed to work anyway?

Reply:
yup, this was poor copy on my part. think i committed wrong as i initially had the tests passing locally. fixed now